demo/demo-spring-boot-transport/demo-spring-boot-pojo-server/src/main/java/io/servicecomb/handler/MyHandler.java

```diff
@@ -0,0 +1,31 @@
+package io.servicecomb.handler;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.servicecomb.core.Handler;
+import io.servicecomb.core.Invocation;
+import io.servicecomb.demo.server.User;
+import io.servicecomb.swagger.invocation.AsyncResponse;
+
+public class MyHandler implements Handler {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MyHandler.class);
+
+  public static final String SPLITPARAM_RESPONSE_USER_SUFFIX = "(modified by MyHandler)";
+
+  @Override
+  public void handle(Invocation invocation, AsyncResponse asyncResp) throws Exception {
+    LOGGER.info("If you see this log, that means this demo project has been converted to ServiceComb framework.");
+
+    invocation.next(response -> {
+      if (invocation.getOperationName().equals("splitParam")) {
```

Comment:
schema+operation is a unique key
OperationMeta have some qualifiedName method, you can call one of them.

Reply:
```java
    invocation.next(response -> {
      if (invocation.getOperationMeta().getSchemaQualifiedName().equals("server.splitParam")) {
```
the judgement logic has been changed like above. Is this what you mean?